blog/2023-12-07-announcing-zoe-cooper.md

```diff
@@ -1,7 +1,7 @@
 ---
 author: Martin Magdinier
 title: Announcing Zoe Cooper as OpenRefine Lead Designer
-slug: 2023/12/07/announcing-zoe-cooper.md
+slug: 2023/12/07/announcing-zoe-cooper.html
```

Comment:
Changing the slug changes the URL of the page, which creates dead links. In the future, can we stick to not using any `slug` at all, letting Docusaurus generate the URL by itself? It would avoid this sort of problem…